### DISCUSSION TITLE
Tips for reverse engineering ps1 games

### DISCUSSION THREAD

**Discussion by JorgeNachtigall** [OverSurge/PS1-Argonaut-Reverse#8] [Q&A] [answered]
Hi!

First of all, congrats on making this project. It's amazing to see a amazing project and so well documented likes this =)

I'm looking for tips to start reverse engineering ps1 games. @OverSurge could you share with me your workflow/tools or any guidance that you used for doing this work?

Thanks in advice.
Cheers!

**Comment by bartlomiejduda** [comment, marked as answer]
Just go here https://forum.xentax.com/viewtopic.php?f=29&t=22266

**Comment by OverSurge** [comment]
Hi :)

I didn't know much about how to do a proper reverse at the time I did this, I was basically tinkering with the game's ISO, boot it into an emulator and write down what happened in order to understand the structure of the game files.

You will probably learn a lot more reading the tutorials @bartlomiejduda sent.

Feel free to ask me specific questions if you have some ;)

**Comment by JorgeNachtigall** [comment]
Hi!

Thank you both @OverSurge and @bartlomiejduda ! Amazing start content provided by @bartlomiejduda 😄 And thank you @OverSurge for you disponibility!

I think that I've a lot to read and start poking around now!

Cheers!
